picard.spec

```diff
@@ -85,8 +100,17 @@ coll = COLLECT(exe,
 
 
 if platform.system() == 'Darwin':
-    info_plist = {'NSHighResolutionCapable': 'True',
-                  'NSPrincipalClass': 'NSApplication'}
+    info_plist = {
+        'NSHighResolutionCapable': 'True',
+        'NSPrincipalClass': 'NSApplication',
+        'CFBundleName': 'picard',
```

Comment:
Must be `Picard` (uppercase P), because this will be used as displayed name of the macOS menu

Reply:
Done.